src/main/java/seedu/address/storage/JsonSerializableAddressBook.java

```diff
@@ -11,24 +11,24 @@
 import seedu.address.commons.exceptions.IllegalValueException;
 import seedu.address.model.AddressBook;
 import seedu.address.model.ReadOnlyAddressBook;
-import seedu.address.model.person.Person;
+import seedu.address.model.client.Client;
 
 /**
  * An Immutable AddressBook that is serializable to JSON format.
  */
 @JsonRootName(value = "addressbook")
 class JsonSerializableAddressBook {
 
-    public static final String MESSAGE_DUPLICATE_PERSON = "Persons list contains duplicate person(s).";
+    public static final String MESSAGE_DUPLICATE_CLIENT = "Clients list contains duplicate Client(s).";
 
-    private final List<JsonAdaptedPerson> persons = new ArrayList<>();
+    private final List<JsonAdaptedClient> Clients = new ArrayList<>();
 
     /**
-     * Constructs a {@code JsonSerializableAddressBook} with the given persons.
+     * Constructs a {@code JsonSerializableAddressBook} with the given Clients.
      */
     @JsonCreator
-    public JsonSerializableAddressBook(@JsonProperty("persons") List<JsonAdaptedPerson> persons) {
-        this.persons.addAll(persons);
+    public JsonSerializableAddressBook(@JsonProperty("Clients") List<JsonAdaptedClient> clients) {
+        this.Clients.addAll(clients);
```

Comment:
change @JasonProperty("Clients") to @JasonProperty("clients")

Reply:
there will be 5 test case errors because the json files are all Clients instead of clients if I'm not wrong

Reply:
Yes. I expect it to have more errors. 
Because, the actual data file is saved as caps Client instead of client.

I can only help u find these files, that you might be likely to change them from caps C to small C. 

![image](https://user-images.githubusercontent.com/14960144/93977686-7148fe00-fdad-11ea-8ab7-754c6c43ac89.png)

Reply:
what do you mean by the actual data file?